fairlearn/metrics/_extra_metrics.py

```diff
@@ -215,3 +215,22 @@ def false_negative_rate(y_true,
 def _root_mean_squared_error(y_true, y_pred, **kwargs):
     r"""Calculate the root mean squared error."""
     return skm.mean_squared_error(y_true, y_pred, squared=False, **kwargs)
+
+
+def count(y_true, *args, **kwargs) -> int:
+    r"""Calculate the number of data points in each group when working with `MetricFrame`.
+
+    The ``y_true`` argument is used to make this calculation. For consistency with
+    other metric functions, the ``y_pred`` argument is required, but ignored.
```

Comment:
This doesn't match the argument list above....

Reply:
Given the pattern shared by other metrics, I'd be inclined to include `y_pred` as the only other argument (my comment above was directed at `sample_weight`). You can check that `y_pred` and `y_true` have the same length before returning.

Reply:
Thank you for this feedback. I made the changes.